### ISSUE TITLE
cmd/tailscaled: support operator-user with FreeIPA 

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#12601]
We got a report that`--operator-user` doesn't work with FreeIPA (Kerberos, it seems)

> error looking up operator "jeff" uid: user: unknown user jeff.

/cc @timtailscale

**Comment by villapx-path** [comment]
Oh wow, this is (I think) exactly what I'm just now running into.

We use FreeIPA/Kerberos for our Linux accounts, and we're mostly on Ubuntu 20.04; I can run some tests if you need any help debugging.

For additional context, my issue is that I'm trying to use `sudo tailscale up --operator=$(whoami) --accept-routes` so that I can use `tailscale file cp` without `sudo`.

However, neither the `sudo tailscale up --operator=$(whoami)` nor `sudo tailscale set --operator=$(whoami)` allow me to do this:

```shell
[13:36] jvilla@004e01fc442e:~ $ sudo tailscale set --operator=$(whoami)
[13:36] jvilla@004e01fc442e:~ $ tailscale file cp ./README.md path-hpwnjx3:
README.md                                  220.00B       152.49B/s    100.00%    ETA 00:00:00
Access denied: file access denied

Use 'sudo tailscale file cp ./README.md path-hpwnjx3:' or 'tailscale up --operator=$USER' to not require root.
```

The tailscale socket is still owned by `root:root`, without others-write permission, hence why I can't write to it:

```shell
[13:37] jvilla@004e01fc442e:~ $ ls -l /run/tailscale/tailscaled.sock
srw-rw-rw- 1 root root 0 Jun 25 13:24 /run/tailscale/tailscaled.sock
```

**Comment by bradfitz** [comment]
The `/run/tailscale/tailscaled.sock` permissions aren't the problem. Anybody can connect/write to that on Linux. The daemon uses peer creds to look up the identity of the caller. The problem was that we ship static builds without libc/NSS user lookups. I just merged #12602 which should fix it. Want to try that out? Or you can wait for the next unstable build.

**Comment by villapx-path** [comment]
Nice, that worked like a charm. Good work!

```shell
[14:20] jvilla@004e01fc442e:~ $ tailscale file cp ./README.md path-hpwnjx3:
README.md                                  220.00B       151.48B/s    100.00%    ETA 00:00:00
```
